docs/api_versioning.rst

```diff
@@ -0,0 +1,48 @@
+API Versioning
+===============
+
+All current APIs are prefixed with:
+
+    /api/1.1
+
+RackHD extenders can supplement the central API (common) with versioned customer-specific APIs in parallel.
+
+Referencing API Versions in URIs
+--------------------------------
+
+    /api/current/...
+    /api/1.1/...
+
+The second /[...]/ block in the URI is the version number. The "current" or "latest" placeholder points to the latest version of the API in the system.
+
+Multiple API versions can be added in parallel. Use N, N-1, N-2, etc. as the naming convention.
+
+All API versioning information should be conveyed in HTTP headers.
+
+Versioning Resources
+---------------------------------------------------------
+
+A translation and validation chain is used to support versioned "types" for URI resources from the RackHD system. The chain flow is:
+
+    +-----------------+    +-----------+    +----------+
+    | business object | -- | translate | -- | validate |
```

Comment:
if you want to keep this as a preformatted (fixed width) text block, I think you can add

```.. code-block: REST```

with a newline above this text, and it'll render as a fixed width segment